sale_stock_picking_blocking/views/sale_stock_picking_blocking_reason_view.xml

```diff
@@ -5,13 +5,13 @@
 AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).-->
 <odoo>
     <record id="sale_stock_picking_blocking_reason_tree_view" model="ir.ui.view">
-        <field name="name">sale.delivery.block.reason.tree</field>
+        <field name="name">sale.delivery.block.reason.view_list</field>
```

Comment:
Maybe there was some confusion due to the discussion about guidelines, but please keep the original view xml id.